oh_staff_ui/templates/oh_staff_ui/search_results.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'oh_staff_ui/base.html' %}
+
+{% block content %}
+{% if results %}
+<table style="width: 100%">
```

Comment:
Let's avoid the use of inline CSS.  Instead, move styles (here & lines 12-13) to a new file `static/css/style.css`, similar to https://github.com/UCLALibrary/dlcs-staff-ui/blob/main/static/css/style.css , and uncomment line 6 in `oh_staff_ui/templates/oh_staff_ui/base.html`.